src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { ErrorMessages } from '../../types/ErrorMessages';
+
+type Props = {
+  message: string;
+  setErrorMessage: (message: ErrorMessages) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  message,
+  setErrorMessage,
+}) => {
+  return (
```

Comment:
```suggestion
}) => (
```

can be a bit simplify